### DISCUSSION TITLE
What is Whisky exactly, is it better than crossover 22.1.1?

### DISCUSSION THREAD

**Discussion by sussyboiiii** [Whisky-App/Whisky#99] [Q&A] [answered]
Hello, 

I have just stumbled upon this and am now wondering, does this project add more compatibility and fixes ontop of what crossover already does or what are the advantages of this project?

And due to immigration of GPT does it work on Ventura as good as on Sonoma?

Thanks!

**Comment by IsaacMarovitz** [comment, marked as answer]
Whisky uses GPT under the hood.

GPT uses CrossOver 22.1.1 under the hood.

CrossOver uses Wine under the hood.

Basically, Whisky is a nice UI wrapper for Wine (like CrossOver) that makes the setup process a lot easier. Right now, you're likely to have better luck with compatibility when using CrossOver. They have a complicated patching system called [CrossTies](https://www.codeweavers.com/support/wiki/c4p_main) which Whisky currently doesn't have an equivalent for.

The difference between CrossOver and Whisky is that since we use GPT under the hood, we can support DirectX12 games that CrossOver can't right now, like Elden Ring, Cyberpunk 2077 etc. If you're running a game that doesn't use DirectX12, CrossOver is likely going to be a better bet.

As for GPT's functionality on Ventura. Ventura is missing some Metal features that GPT might make use of, so it's possible there may be some holes in its compatibility, but for the most part it works okay.
